ui/components/app/permissions-connect-permission-list/index.scss

```diff
@@ -0,0 +1,22 @@
+.permissions-connect-permission-list {
+  .permission {
+    @include H6;
+
+    width: 100%;
+    padding-bottom: 16px;
+    border-bottom: 1px solid $Grey-100;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    color: $Black-100;
+
+    i {
+      display: block;
+      padding: 16px;
+      min-width: 16px;
+      min-height: 16px;
```

Comment:
Is this so if the font-awesome class is not provided the layout remains the same?

Reply:
Exactly